src/Utils/Hooks/useFulfillmentOptions.ts

```diff
@@ -0,0 +1,156 @@
+import { useFragment } from "react-relay"
+import { useFulfillmentOptions_artwork$key } from "__generated__/useFulfillmentOptions_artwork.graphql"
+import { useFulfillmentOptions_me$key } from "__generated__/useFulfillmentOptions_me.graphql"
+import { graphql } from "relay-runtime"
+
+// import { EU_COUNTRY_CODES } from "Components/CountrySelect"
+
+const EU_COUNTRY_CODES = [
+  "AT",
+  "BE",
+  "BG",
+  "HR",
+  "CY",
+  "CZ",
+  "DK",
+  "EE",
+  "FI",
+  "FR",
+  "DE",
+  "GR",
+  "HU",
+  "IE",
+  "IT",
+  "LV",
+  "LT",
+  "LU",
+  "MT",
+  "NL",
+  "PL",
+  "PT",
+  "RO",
+  "SK",
+  "SI",
+  "ES",
+  "SE",
+] as const
+
+type MPMoney = {
+  minor: any
+}
+
+type ShippingUnion = MPMoney | "ARTSY_SHIPPING" | null
+
+type FulfillmentOptions = {
+  artworkLocation: {
+    countryCode: string
+  }
+  pickupAvailable: boolean
+  onlyShipsDomestically: boolean
+  domesticShipping: ShippingUnion
+  internationalShipping: ShippingUnion
+  freeGlobalShipping: boolean
+  freeShippingToUserCountry?: boolean
+}
+
+export const useFulfillmentOptionsForArtwork = (
```

Comment:
@erikdstock - We've gone over this a few times in the past, but to rehash that discussion, we should avoid using hooks to fetch relay data in this way unless strictly necessary. Its not the worst thing in the world, but its a uncommon idiom and tricky due to data masking at the props / component level

Reply:
If we really implemented this feature I think we would move the data loading up into a MP type or even gravity fields. I did it here specifically for hackathon POC purposes and to have the option of quick collaboration with the work @oxaudo is doing to simplify some incorrect shipping messaging - I wanted a helper to encapsulate that logic in one place, and for temporary code this seemed better and more readable than writting a mirror of a graphql type.

If it becomes clear that we want to actually implement something like this or if I'm able to finish another cleanup PR i have in review I'll come back to this but regardless I'll nothing like this will merge.